pandas/_libs/src/vendored/ujson/python/JSONtoObj.c

```diff
@@ -40,7 +40,10 @@ Numeric decoder derived from TCL library
 
 #include "pandas/vendored/ujson/lib/ultrajson.h"
```

Comment:
Can you remove the include on L41 now? 

I'm assuming the issue is that the Python include is supposed to come first

Reply:
> Can you remove the include on L41 now?

Yes, of course. Seems I missed that in between creating the issue, testing and creating the PR. Pushed a new commit.